From 78079a80858fadb6c3eb0940166c5666dded2535 Mon Sep 17 00:00:00 2001
From: Jakob Schiotz <schiotz@fysik.dtu.dk>
Date: Mon, 6 Jul 2026 12:41:56 +0200
Subject: [PATCH] Update EasyBuild config file for Intel compilers.

Update the mechanism for specifying the MPI compiler used by GPAW.  Also switch
to the C++ compiler as GPAW is now compiled as C++ per default.

---
 doc/platforms/Linux/EasyBuild/config_intel.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/platforms/Linux/EasyBuild/config_intel.py b/doc/platforms/Linux/EasyBuild/config_intel.py
index 125ca3bfc3..64b49fc8d6 100644
--- a/doc/platforms/Linux/EasyBuild/config_intel.py
+++ b/doc/platforms/Linux/EasyBuild/config_intel.py
@@ -1,4 +1,5 @@
-mpicompiler = 'mpiicc'
+mpi = True
+compiler = 'mpiicpx'
 
 # FFTW should be configured from environment variables, but they do
 # not report the correct names for a dynamically loaded library.
-- 
2.43.7

